lib/public_key/src/public_key.erl

```diff
@@ -465,26 +464,31 @@ pem_entry_encode('SubjectPublicKeyInfo', Entity=#'RSAPublicKey'{}) ->
                                                           parameters =?DER_NULL}, KeyDer),
     pem_entry_encode('SubjectPublicKeyInfo', Spki);
 pem_entry_encode('SubjectPublicKeyInfo',
-                 {DsaInt, Params=#'Dss-Parms'{}}) when is_integer(DsaInt) ->
+                 {DsaInt, Params=#'DSA-Params'{}}) when is_integer(DsaInt) ->
```

Comment:
Same here doesn't we translate to Dss-Parms in the API, I'm confused, or did we miss translation somewhere.